templates/base.html

```diff
@@ -37,15 +38,15 @@
         {% include "includes/footer.html" %}
     </footer>
 
-<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
-    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous">
-</script>
+<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
     integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous">
 </script>
+ <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>

```

Comment:
Can you provide some examples of the use of jQueryUI used on this project? Is this needed?

Reply:
we changed from jquery to bootstrap.